packages/design-system/src/components/N8nInfoTip/InfoTip.vue

```diff
@@ -84,7 +84,7 @@ const iconData = computed((): { icon: string; color: string } => {
 				</span>
 			</template>
 		</N8nTooltip>
-		<span v-else :class="$style.iconText">
+		<span v-else :class="$style.iconText" :style="{ color: iconData.color }">
```

Comment:
This is potentially new behavior for other existing notes. It's required to provide the color to a InfoTip without Tooltip.
This seems like more of an oversight since these properties are defined together, but we may want to avoid this change regardless.

Please advise whether we're happy with this, should move this behavior behind a parameter or keep the change to RunInfo and try to change the color from there, duplicating code in the process.

Reply:
Good catch, seems like it's an oversight since it's used in the tooltip icon [here](https://github.com/n8n-io/n8n/blob/ad13a756d7f90541705107c033379db471c17925/packages/design-system/src/components/N8nInfoTip/InfoTip.vue#L78).
Regarding the code duplication question: Are you worried about the fact that now we have duplicate set of attributes [here](https://github.com/n8n-io/n8n/blob/5bb70633abfccffbdd7fb992505cccc4feb333b4/packages/design-system/src/components/N8nInfoTip/InfoTip.vue#L78) and here? This is fine by me but we don't need to conditionally render the tooltip here, it already has the `disabled` property set up so it won't show if `type !== tooltip` so the whole component can be simplified as:
```vue
<template>
	<div
		:class="{
			'n8n-info-tip': true,
			[$style.infoTip]: true,
			[$style[theme]]: true,
			[$style[type]]: true,
			[$style.bold]: bold,
		}"
	>
		<N8nTooltip
			:placement="tooltipPlacement"
			:popper-class="$style.tooltipPopper"
			:disabled="type !== 'tooltip'"
		>
			<span :class="$style.iconText" :style="{ color: iconData.color }">
				<N8nIcon :icon="iconData.icon" />
			</span>
			<template #content>
				<span>
					<slot />
				</span>
			</template>
		</N8nTooltip>
	</div>
</template>
````

Reply:
Also, we can maybe just send the color straight to the `N8nIcon` component but we'll need to update the computed property so the color is in the format this component expects it.

Reply:
Not related to this PR, but can we also fix color specification [here](https://github.com/n8n-io/n8n/blob/5bb70633abfccffbdd7fb992505cccc4feb333b4/packages/design-system/src/components/N8nInfoTip/InfoTip.vue#L31) to `var(--color-text-light)`